sdk/src/main/java/com/sygic/travel/sdk/StSDK.kt

```diff
@@ -5,86 +5,91 @@ import android.content.Context
 import com.sygic.travel.sdk.model.place.Place
 import com.sygic.travel.sdk.model.place.Tour
 import com.sygic.travel.sdk.model.query.PlacesQuery
 import com.sygic.travel.sdk.model.query.ToursQuery
+import com.sygic.travel.sdk.provider.DataProvider
 import io.reactivex.disposables.Disposable
 import io.reactivex.disposables.Disposables
-import com.sygic.travel.sdk.provider.DataProvider
 import java.io.File
 
+
 /**
  * Provides public methods for requesting API.
  */
-class StSDK internal constructor() {
-
-    /**
-     * Creates and sends a request to get places, e.g. for map or list.
-     * @param placesQuery PlacesQuery encapsulating data for API request.
-     * *
-     * @param back Callback. Either [Callback.onSuccess] with places is called, or
-     * *             [Callback.onFailure] in case of an error is called.
-     */
-    fun getPlaces(placesQuery: PlacesQuery, back: Callback<List<Place>?>?) {
-        disposable = dataProvider?.getPlaces(placesQuery, back)!!
-    }
-
-
-    /**
-     *
-     * Creates and sends a request to get place with detailed information.
-     * @param id Unique id of a place - detailed information about this place will be requested.
-     * *
-     * @param back Callback. Either [Callback.onSuccess] with place is called, or
-     * *             [Callback.onFailure] in case of an error is called.
-     */
-    fun getPlaceDetailed(id: String, back: Callback<Place?>) {
-	    disposable = dataProvider?.getPlaceDetailed(id, back)!!
-    }
-
-
-    /**
-     *
-     * Creates and sends a request to get places with detailed information.
-     * @param ids Ids of places - detailed information about these places will be requested.
-     * *
-     * @param back Callback. Either [Callback.onSuccess] with places is called, or
-     * *             [Callback.onFailure] in case of an error is called.
-     */
-    fun getPlacesDetailed(ids: List<String>, back: Callback<List<Place>?>) {
-        disposable = dataProvider?.getPlacesDetailed(ids, back)!!
-    }
-
-
-    /**
-     *
-     * Creates and sends a request to get the place's media.
-     * @param id Unique id of a place - media for this place will be requested.
-     * *
-     * @param back Callback. Either [Callback.onSuccess] with places is called, or
-     * *             [Callback.onFailure] in case of an error is called.
-     */
-    fun getPlaceMedia(id: String, back: Callback<List<Medium>?>) {
-        disposable = dataProvider?.getPlaceMedia(id, back)!!
-    }
-
-
-    /**
-     * Creates and sends a request to get the Tours.
-     * @param toursQuery ToursQuery encapsulating data for API request.
-     *
-     * @param back Callback. Either [Callback.onSuccess] with tours is called, or
-     *            [Callback.onFailure] in case of an error is called.
-     */
-    fun getTours(toursQuery: ToursQuery, back: Callback<List<Tour>?>?) {
-        disposable = dataProvider?.getTours(toursQuery, back)!!
-    }
+class StSDK {
+
+	/**
```

Comment:
change of coding style should be separated from this PR.

Reply:
:heavy_check_mark: